### PR TITLE
doc(release): Prepare release notes for oss 23.04

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Prepare release notes for:
* WEB 23.04.8

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [x] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 1]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
 ## Centreon Web
 
+### 23.04.8
+
+Release date: `September 21, 2023`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Core] Added feature flag system
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:32]:
```suggestion
- [Core] Added feature flag system.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 2]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Configuration] Added feature flag to Vault Configuration endpoints.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:33]:
```suggestion
- [Configuration] Added feature flag to vault configuration endpoints.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 3]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [authentication] Removed password fields if contact is not local type.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:34]:
```suggestion
- [Authentication] Removed password fields from profile page for accounts using delegated authentication.	
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 4]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+
+Release date: `September 21, 2023`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Core] Added feature flag system
+- [Configuration] Added feature flag to Vault Configuration endpoints.
+- [authentication] Removed password fields if contact is not local type.	
+- [Anomaly Detection] Added a feature flag to hide exclusion of periods.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:35]:
```suggestion
- [Anomaly Detection] Added a feature flag to hide the period exclusion interface.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 5]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+
+</details>
+
+<details open>
+  <summary>Bug fixes</summary>
+
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Don't list Business Activities as service in configuration form.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:43]:
```suggestion
- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 6]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [CLAPI] Fixed the export of severities to be able to reimport them.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:44]:
```suggestion
- [CLAPI] Fixed the export of severities to make it possible to reimport them.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 7]
```diff
@@ -22,6 +22,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:46]: unclear

**Review by rsadik-ext — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 8]
```diff
@@ -22,6 +22,33 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+
+<details open>
+  <summary>Bug fixes</summary>
+
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
+- [CLAPI] Fixed the export of severities to make it possible to reimport them.
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).	
+- Fixed SLQi in centreonLogAction.class.php.
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:47]:
```suggestion
- [Security] Fixed SLQi in centreonLogAction.class.php.
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 9]
```diff
@@ -23,6 +23,33 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+
+<details open>
+  <summary>Bug fixes</summary>
+
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Don't list Business Activities as service in configuration form.
+- [CLAPI] Fixed the export of severities to be able to reimport them.
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).	
+- Fixed SLQi in centreonLogAction.class.php.
```
Comment (lpinsivy) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:48]:
```suggestion
- [Security] Fixed SLQi in centreonLogAction.class.php.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 10]
```diff
@@ -22,6 +22,33 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+</details>
+
+<details open>
+  <summary>Bug fixes</summary>
+
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
+- [CLAPI] Fixed the export of severities to make it possible to reimport them.
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:46]:
```suggestion
- [Authentication] Fixed the access to Centreon UI when a default page is configured for the user and authentication uses SSO (OIDC/SAML).	
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 11]
```diff
@@ -23,6 +23,33 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
 ## Centreon Web
 
+### 23.04.8
+
+Release date: `September 21, 2023`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Core] Added feature flag system
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:33]:
```suggestion
- [Core] Added feature flag system.
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 12]
```diff
@@ -23,6 +23,33 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Configuration] Added feature flag to Vault Configuration endpoints.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:34]:
```suggestion
- [Configuration] Added feature flag to vault configuration endpoints.
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 13]
```diff
@@ -23,6 +23,33 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [authentication] Removed password fields if contact is not local type.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:35]:
```suggestion
- [Authentication] Removed password fields from profile page for accounts using delegated authentication.
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 14]
```diff
@@ -23,6 +23,33 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+
+Release date: `September 21, 2023`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Core] Added feature flag system
+- [Configuration] Added feature flag to Vault Configuration endpoints.
+- [authentication] Removed password fields if contact is not local type.	
+- [Anomaly Detection] Added a feature flag to hide exclusion of periods.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:36]:
```suggestion
- [Anomaly Detection] Added a feature flag to hide the period exclusion interface.
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 15]
```diff
@@ -23,6 +23,33 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Don't list Business Activities as service in configuration form.
+- [CLAPI] Fixed the export of severities to be able to reimport them.
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).	
+- [Security] Fixed SLQi in centreonLogAction.class.php.
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Resources Status] Fixed filter on hostgroups and categories that doesn't returning hosts.
+- [Configuration] Fixed custom macro inheritance for hosts.
+
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:52]:
```suggestion
- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
- [CLAPI] Fixed the export of severities to make it possible to reimport them.
- [CLAPI] Removed the possibility to link a regular resource with a template.	
- [Authentication] Fixed access to the Centreon UI when a default page is configured for the user and authentication uses SSO (OIDC/SAML).	
- [Security] Fixed SLQi in centreonLogAction.class.php.
- [Resources Status] Fixed filters on hostgroups and categories that weren't returning hosts.
- [Configuration] Fixed custom macro inheritance for hosts.

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 16]
```diff
@@ -22,6 +22,33 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
 
 ## Centreon Web
 
+### 23.04.8
+
+Release date: `September 21, 2023`
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:27]:
```suggestion
Release date: `September 22, 2023`
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 17]
```diff
@@ -23,6 +23,32 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
 
 ## Centreon Web
 
+### 23.04.8
+
+Release date: `September 21, 2023`
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:28]:
```suggestion
Release date: `September 22, 2023`
```

**Review by cg-tw — APPROVED** [review]

**Review by rsadik-ext — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 18]
```diff
@@ -22,6 +22,33 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+  <summary>Enhancements</summary>
+
+- [Core] Added feature flag system.
+- [Configuration] Added feature flag to vault configuration endpoints.
+- [Authentication] Removed password fields from profile page for accounts using delegated authentication.	
+- [Anomaly Detection] Added a feature flag to hide the period exclusion interface.
+
+</details>
+
+<details open>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:39]:
```suggestion
<details>
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 19]
```diff
@@ -22,6 +22,33 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
+- [CLAPI] Fixed the export of severities to make it possible to reimport them.
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed access to the Centreon UI when a default page is configured for the user and authentication uses SSO (OIDC/SAML).	
+- [Security] Fixed SLQi in centreonLogAction.class.php.
+- [Resources Status] Fixed filters on hostgroups and categories that weren't returning hosts.
+- [Configuration] Fixed custom macro inheritance for hosts.
+
+</details>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:51]:
```suggestion
</details>

```

**Review by lpinsivy — DISMISSED** [review]

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 20]
```diff
@@ -23,6 +23,32 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+  <summary>Enhancements</summary>
+
+- [Core] Added feature flag system.
+- [Configuration] Added feature flag to vault configuration endpoints.
+- [Authentication] Removed password fields from profile page for accounts using delegated authentication.
+- [Anomaly Detection] Added a feature flag to hide the period exclusion interface.
+
+</details>
+
+<details open>
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:40]:
```suggestion
<details>
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 21]
```diff
@@ -23,6 +23,32 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
+- [CLAPI] Fixed the export of severities to make it possible to reimport them.
+- [CLAPI] Removed the possibility to link a regular resource with a template.	
+- [Authentication] Fixed access to the Centreon UI when a default page is configured for the user and authentication uses SSO (OIDC/SAML).	
+- [Security] Fixed SLQi in centreonLogAction.class.php.
+- [Resources Status] Fixed filters on hostgroups and categories that weren't returning hosts.
+- [Configuration] Fixed custom macro inheritance for hosts.
+
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:51]:
```suggestion

</details>

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by lpinsivy — APPROVED** [review]

**Pull request merged**